.github/workflows/triage-prs.yaml

```diff
@@ -12,7 +12,8 @@ permissions:
 jobs:
   pull-requests-comment:
     name: Comment and Triage
-    if: github.repository == 'kanisterio/kanister' && github.actor != 'dependabot'
+    # The entire job is skipped for @dependabot
+    if: github.repository == 'kanisterio/kanister' && github.actor != 'dependabot[bot]'
```

Comment:
Got `dependabot[bot]` from the examples in the documentation, not 100% sure this is correct. We'll know after merging.